### ISSUE TITLE
Voice message replies auto opens a new reply

### ISSUE THREAD

**Issue by ILoveSkilledCoders** [telegramdesktop/tdesktop#25341]
### Steps to reproduce

After replying to a comment with voice message, TG controls my computer and starts the reply process to that message again.

I reply to someone using VM and after I send it, TG does a reply to, to the same message again so the text field is waiting for me to do a VM or type. This has been going on for days now since your first update & it's very frustrating.

Someone else said he's also having that problem.

### Expected behaviour

I reply to a message with a VM & it no longer shows that I'm trying to reply to a message.

### Actual behaviour

See above

### Operating system

8.1

### Version of Telegram Desktop

4.3.1

### Installation source

Static binary from official website

**Comment by 23rd** [comment]
Unfortunately I can't reproduce, can you share a demonstration video?

**Comment by ILoveSkilledCoders** [comment]
> Unfortunately I can't reproduce, can you share a demonstration video?

So when I tried to do a video for you guys a few days ago I couldn't get the bug to reproduce.

Then just now without the video I tried it, again, couldn't get it to reproduce, but then when I left a long message it happened again, then when I left over a minute message again, it happened, so I think when it's a 10 sec test VM it doesn't do that, but I'm pretty sure I've done tests before for that short & it happened or it's becoming intermittent or only with some accounts, but earlier this week it was happening with everyone.

And there's no way I'm going to record a video of me leaving a message for someone for 18 mins. 

I'll see if I can try it with another account later on.

Thanks

**Comment by ILoveSkilledCoders** [comment]
> Unfortunately I can't reproduce, can you share a demonstration video?

Ok, I finally managed to get a video, but can I PM it to you please. I don't want it public, it has people's comments in it.

**Comment by VetaLioSTV** [comment]
@23rd could you check my [post ](https://github.com/telegramdesktop/tdesktop/issues/25335#issuecomment-1312067663) also? I have the same issue

**Comment by 23rd** [comment]
@ILoveSkilledCoders You can privately send a video to t.me/preston.

**Comment by ILoveSkilledCoders** [comment]
> @ILoveSkilledCoders You can privately send a video to t.me/preston.

I messaged you, saw that you & I had talked once before, messaged to confirm you are tech support for TG & you ignored me, then I tried to message you again & got this. https://i.imgur.com/n1h6Zsr.png did you just mark me as a spammer for no reason?

**Comment by john-preston** [comment]
@ILoveSkilledCoders this is my account, I have no idea if I blocked you, sorry.

What is your username? Let me unblock several last blocked users 🙈

**Comment by ILoveSkilledCoders** [comment]
> @ILoveSkilledCoders this is my account, I have no idea if I blocked you, sorry.
> 
> What is your username? Let me unblock several last blocked users see_no_evil

@ourfreesociety

But why does you blocking me put me in TG jail? That's not fair. :(

And all I did was ask you if you were from TG tech support & you could see we talked once before, why would you block me?

**Comment by ilya-fedin** [comment]
@ILoveSkilledCoders he's not tech support, he's tdesktop developer and he can't ban you globally

**Comment by ILoveSkilledCoders** [comment]
> @ILoveSkilledCoders he's not tech support, he's tdesktop developer and he can't ban you globally

Well what I've come to learn is that if even one person does something to my profile, it puts me in TG jail.

**Comment by VetaLioSTV** [comment]
@ilya-fedin as i said, i have identical problem. Just answer with voice message on received voice message, it should be easily replicated.

**Comment by ILoveSkilledCoders** [comment]
So... I replaced my HD with an SSD HD & also went back to Win 7 from Win 8.1

The volume issues are fixed, BUT this issue still remains.

And b/c I isn't rush to turn off auto updates in settings, it updated to the latest version. 

I wanted to see if the older version was doing it b/c it wasn't when I was on it.

**Comment by ILoveSkilledCoders** [comment]
> @ilya-fedin as i said, i have identical problem. Just answer with voice message on received voice message, it should be easily replicated.

I already figured out that it only happens after the 1 minute mark & I told them that above.

**Comment by ILoveSkilledCoders** [comment]
So it doesn't seem like anyone is going to fix this HUGE bug.

I DID confirm that it only started happening after the update after this version - tsetup-x64.4.1.0 - or maybe one after that when I first complained.

I had to redo my Windows 3 times & I rushed to turn off auto updates on TG the very last time & I never had the problem, then I had to log into TG again & the update took place & again, back to this bug.

It happened on both Win 8.1 & Win 7.

PLEASE fix it.

**Comment by VetaLioSTV** [comment]
> 



> So it doesn't seem like anyone is going to fix this HUGE bug.
> 
> I DID confirm that it only started happening after the update after this version - tsetup-x64.4.1.0 - or maybe one after that when I first complained.
> 
> I had to redo my Windows 3 times & I rushed to turn off auto updates on TG the very last time & I never had the problem, then I had to log into TG again & the update took place & again, back to this bug.
> 
> It happened on both Win 8.1 & Win 7.
> 
> PLEASE fix it.

Agreed. Its super annyoing and still no reaction at all from Telegram team. @ilya-fedin any thoughts??

**Comment by ilya-fedin** [comment]
As far as I understand, neither @john-preston or @23rd can reproduce the issue and since @ILoveSkilledCoders got a spam block, his messages haven't reached @john-preston in Telegram, so the issue has stalled.

**Comment by VetaLioSTV** [comment]
@ilya-fedin OK, can i raise a new standalone issue with gif animation? This is ultra annyoing, approx month with that issue, PC desktop client became unusable..

**Comment by ilya-fedin** [comment]
I don't see any reason to create a duplicate issue, this won't help at all

**Comment by ilya-fedin** [comment]
I guess there's not so much people affected (or there were way more participants in the issue), so the issue is treated as a minor one and developers are currently busy with really prioritized tasks (features for the next release).

**Comment by VetaLioSTV** [comment]
> I don't see any reason to create a duplicate issue, this won't help at all

As far as i understood -> duplicating of issue makes more sense, since telegram team fixes it with 1-2 day (as it was with archived chats and UI bug here-> #25413 ,#25408 and [here](https://github.com/telegramdesktop/tdesktop/issues/25408#issuecomment-1324330818)

**Comment by ilya-fedin** [comment]
Those don't look like duplicates or one of them would be closed as a duplicate

**Comment by VetaLioSTV** [comment]
> I guess there's not so much people affected (or there were way more participants in the issue), so the issue is treated as a minor one and developers are currently busy with really prioritized tasks (features for the next release).

That's the problem. I think it has a reference to Win 8.1... and what are we left to do?

**Comment by ilya-fedin** [comment]
Well, you can try to do what @ILoveSkilledCoders can't do due to the spam ban?

**Comment by VetaLioSTV** [comment]
> As far as I understand, neither @john-preston or @23rd can reproduce the issue and since @ILoveSkilledCoders got a spam block, his messages haven't reached @john-preston in Telegram, so the issue has stalled.

The problem is that @ILoveSkilledCoders didn't upload video example of it here, so he decided to send it directly to @john-preston and received spam block.



> Well, you can try to do what @ILoveSkilledCoders can't do due to the spam ban?

I would like to raise a new issue and this one can be closed i guess?

**Comment by ilya-fedin** [comment]
No, your duplicate one will be closed. Why you so want to create a new one? Just post any relevant details here.

**Comment by VetaLioSTV** [comment]
> No, your duplicate one will be closed. Why you so want to create a new one? Just post any relevant details here.

because its already spammed with non-relevant details.

**Comment by ilya-fedin** [comment]
Well,, you have the choice of posting here or silently waiting

**Comment by VetaLioSTV** [comment]
> Well, you have the choice of posting here or silently waiting

thats not polite...

**Comment by ilya-fedin** [comment]
You can use in-app support if you want polite response. This is a place to discuss issues with open source rules (you want it @ you fix/implement it or silently wait). It's not for end users expecting tech support level response.

**Comment by VetaLioSTV** [comment]
> You can use in-app support if you want polite response. This is a place to discuss issues with open source rules (you want it @ you fix/implement it or silently wait). It's not for end users expecting tech support level response.

1. in-app support doesn't answer at all.
2. we're discussing the issue and you're telling me "you have the choice or silently wait", which sound not polite from a telegram team member(contractor?).

**Comment by ilya-fedin** [comment]
1. Apparently there's no enough volunteers for your account's language.
2. It's the truth, like you it or not. If you want to say something, say it here, you can't create a duplicate. This place is not official to Telegram, by the way. The only official places are the bugs & suggestions platform and in-app support.

**Comment by 23rd** [comment]
As long as there is no any video demonstration, this ticket will be unresolved.

**Comment by ILoveSkilledCoders** [comment]
> 1. Apparently there's no enough volunteers for your account's language.
> 2. It's the truth, like you it or not. If you want to say something, say it here, you can't create a duplicate. This place is not official to Telegram, by the way. The only official places are the bugs & suggestions platform and in-app support.

I already posted it there & as usual, no response.

https://bugs.telegram.org/c/21810

**Comment by ILoveSkilledCoders** [comment]
> As long as there is no any video demonstration, this ticket will be unresolved.

I have the demonstration. Can you give me your TG handle & I'll send it to you?

Thanks

**Comment by ILoveSkilledCoders** [comment]
> > As far as I understand, neither @john-preston or @23rd can reproduce the issue and since @ILoveSkilledCoders got a spam block, his messages haven't reached @john-preston in Telegram, so the issue has stalled.
> 
> The problem is that @ILoveSkilledCoders didn't upload video example of it here, so he decided to send it directly to @john-preston and received spam block.
> 
> > Well, you can try to do what @ILoveSkilledCoders can't do due to the spam ban?
> 
> I would like to raise a new issue and this one can be closed i guess?

It wasn't just that I got blocked, it was that he didn't respond. 

I had spoken to him once before & he ignored me then too.

I'm not going to send a private video to someone who doesn't even respond to PMs to confirm they actually work with/for TG.

**Comment by ILoveSkilledCoders** [comment]
> > I guess there's not so much people affected (or there were way more participants in the issue), so the issue is treated as a minor one and developers are currently busy with really prioritized tasks (features for the next release).
> 
> That's the problem. I think it has a reference to Win 8.1... and what are we left to do?

No, like I said, it's also on Win 7.

AND now when I click on a video, often it goes to a black screen. :(

**Comment by ilya-fedin** [comment]
> I already posted it there & as usual, no response.

Yeah, it's official, but as far as i heard understaffed.

**Comment by VetaLioSTV** [comment]
> No, like I said, it's also on Win 7.

yes, its kinda from Win 7 till Win 8.1 i guess. Can you share video here? Otherwise i can try to make gif animation... Problem is that i wouldn't like to share my personal chat\contact.. so i need to create somekind of fake one.

**Comment by ILoveSkilledCoders** [comment]
> > No, like I said, it's also on Win 7.
> 
> yes, its kinda from Win 7 till Win 8.1 i guess. Can you share video here? Otherwise i can try to make gif animation... Problem is that i wouldn't like to share my personal chat\contact.. so i need to create somekind of fake one.

Like I said further up, I don't want to share the video here for the same reason you don't want to share it LOL b/c it shows my list of people on the left & what was said.

I'm more than happy to share it with a staffer of TG, but in private.

**Comment by ILoveSkilledCoders** [comment]
And this should be tested in groups too to see if you leave a VM after 1 min, it also does the same thing. Let me test it on one of my groups that's dead.

Yes, it does the same thing.

**Comment by ILoveSkilledCoders** [comment]
On another note @VetaLioSTV - how did you emoji one of the comments above? I don't see any way to emoji comments LOL

**Comment by VetaLioSTV** [comment]
@23rd , @john-preston , @Aokromes , @ilya-fedin 
OK, please take a look at demo(i hope for now its much more clear how it looks like):
https://user-images.githubusercontent.com/15155286/205610615-ecaa3535-7e31-4b04-9545-e7950411fb1b.mp4

My suggestion -> let me create a new standalone issue and close that one (?).

PS This the most one ultra annoying bug which is almost reproducible one month+ on desktop client. Please assist...

**Comment by VetaLioSTV** [comment]
@23rd 
at current moment i'm on other environment, so i can't test it. 

@ILoveSkilledCoders could you check at your Win7 and post here a reply?

**Comment by kirsan31** [comment]
I can say that this bug some times happens on win11 and win10 too.

**Comment by ILoveSkilledCoders** [comment]
> @23rd at current moment i'm on other environment, so i can't test it for now.
> 
> @ILoveSkilledCoders could you check at your Win7 and post here a reply?

Test my current version or that link you gave us?

**Comment by ILoveSkilledCoders** [comment]
> I can say that this bug some times happens on win11 and win10 too.

Are you making note that it should be happening after 1 min. of talking?

Although it looks like Veta it happens even under the 1 min. mark.

I'm glad to see more people reporting this HUGE bug.

Thanks

**Comment by VetaLioSTV** [comment]
@23rd , @ILoveSkilledCoders as for version `4.4.1 x64` seems to be its already fixed and i cant reproduce it. Many thanks for fast assist.

**Comment by ILoveSkilledCoders** [comment]
> @23rd , @ILoveSkilledCoders as for version `4.4.1 x64` seems to be its already fixed and i cant reproduce it. Many thanks for fast assist.

@23rd - how do I get the update? Mine is still on the 4.3.3 version.

**Comment by 23rd** [comment]
https://desktop.telegram.org

**Comment by john-preston** [comment]
This is fixed in 4.4.1, you can get it on https://desktop.telegram.org or on https://github.com/telegramdesktop/tdesktop/releases/v4.4.1

**Comment by ILoveSkilledCoders** [comment]
> 



> This is fixed in 4.4.1, you can get it on https://desktop.telegram.org or on https://github.com/telegramdesktop/tdesktop/releases/v4.4.1

Ok, I just realiazed there may be another bug. 

I turned off auto updates & yet it's not telling me there's an update where I can just click the button & it updates instead of me having to DL something.

That's a bug, right?

Also, I was just notified by one of my group members that you guys now have an anti spam feature that DELETES all Truther content from groups & channels, is that correct?
